ow_faults/src/FaultInjector.cpp

```diff
@@ -12,8 +12,8 @@ using namespace ow_lander;
 
 FaultInjector::FaultInjector(ros::NodeHandle node_handle)
 {
-  m_joint_state_sub = node_handle.subscribe("/joint_states", 10, &FaultInjector::jointStateCb, this);
-  m_joint_state_pub = node_handle.advertise<sensor_msgs::JointState>("/faults/joint_states", 10);
+  m_joint_state_sub = node_handle.subscribe("/og_joint_states", 10, &FaultInjector::jointStateCb, this);
+  m_joint_state_pub = node_handle.advertise<sensor_msgs::JointState>("/joint_states", 10); // 'new' joint states topic where fault data is now publishes on
```

Comment:
per ROS CppStyle max line length is 120 characters

Reply:
will adjust